include/script_api/macros.h

```diff
@@ -195,6 +195,8 @@
 /// Blocks for the given number of frames.
 #define EVT_WAIT_FRAMES(NUM_FRAMES)             EVT_CMD(EVT_OP_WAIT_FRAMES, NUM_FRAMES),
 
+#define EVT_WAIT(NUM_FRAMES)             EVT_CMD(EVT_OP_WAIT_FRAMES, NUM_FRAMES),
```

Comment:
Would be better to remove WAIT_FRAMES entirely than just alias I think

Reply:
Will do in a second pass with mass-renaming. Don't want to touch all those files at the moment.